### ISSUE TITLE
Wrong keyword in  "Unit 4: Pre-trained models and datasets for audio classification"

### ISSUE THREAD

**Issue by ritog** [huggingface/audio-transformers-course#55]
In the section [Speech Commands](https://huggingface.co/learn/audio-course/chapter4/classification_models?fw=pt#speech-commands), the code that is supposed to be run is:

```python3
classifier = pipeline(
    "audio-classification", model="MIT/ast-finetuned-speech-commands-v2"
)
classifier(sample["audio"])
```
But it leads to the following error:

```out
ValueError: We expect a numpy ndarray as input
```

<details>
<summary>Full Error Output</summary>

```bash
ValueError                                Traceback (most recent call last)

[<ipython-input-8-a13009e6d325>](https://localhost:8080/#) in <cell line: 4>()
      2     "audio-classification", model="MIT/ast-finetuned-speech-commands-v2"
      3 )
----> 4 classifier(sample["audio"])

3 frames

[/usr/local/lib/python3.10/dist-packages/transformers/pipelines/audio_classification.py](https://localhost:8080/#) in __call__(self, inputs, **kwargs)
    128             - **score** (`float`) -- The corresponding probability.
    129         """
--> 130         return super().__call__(inputs, **kwargs)
    131 
    132     def _sanitize_parameters(self, top_k=None, **kwargs):

[/usr/local/lib/python3.10/dist-packages/transformers/pipelines/base.py](https://localhost:8080/#) in __call__(self, inputs, num_workers, batch_size, *args, **kwargs)
   1118             )
   1119         else:
-> 1120             return self.run_single(inputs, preprocess_params, forward_params, postprocess_params)
   1121 
   1122     def run_multi(self, inputs, preprocess_params, forward_params, postprocess_params):

[/usr/local/lib/python3.10/dist-packages/transformers/pipelines/base.py](https://localhost:8080/#) in run_single(self, inputs, preprocess_params, forward_params, postprocess_params)
   1124 
   1125     def run_single(self, inputs, preprocess_params, forward_params, postprocess_params):
-> 1126         model_inputs = self.preprocess(inputs, **preprocess_params)
   1127         model_outputs = self.forward(model_inputs, **forward_params)
   1128         outputs = self.postprocess(model_outputs, **postprocess_params)

[/usr/local/lib/python3.10/dist-packages/transformers/pipelines/audio_classification.py](https://localhost:8080/#) in preprocess(self, inputs)
    153 
    154         if not isinstance(inputs, np.ndarray):
--> 155             raise ValueError("We expect a numpy ndarray as input")
    156         if len(inputs.shape) != 1:
    157             raise ValueError("We expect a single channel audio input for AutomaticSpeechRecognitionPipeline")

ValueError: We expect a numpy ndarray as input
```
</details>

**Comment by ritog** [comment]
Creating a PR for it now.

**Comment by ritog** [comment]
Created PR #56 that solves this issue.

**Comment by sanchit-gandhi** [comment]
Will be fixed by #58 - I recently updated `transformers` so that the audio classification pipeline now handles resampling automatically under the hood. This should make using the pipeline even easier! These changes are only on the `main` branch, so we need to fetch the latest version of the library from there

**Comment by ritog** [comment]
Thanks for letting me know.
